web/html/src/components/dialog/dialog.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { DeleteDialog } from "./DeleteDialog";
+import { ModalLink } from "./ModalLink";
+
+
+storiesOf('Dialogs', module)
+  .add('delete dialog', () => (
+    <div>
+      <ModalLink
```

Comment:
May be we want to have something more visible to click on

Reply:
![Screenshot from 2019-08-30 11-51-21](https://user-images.githubusercontent.com/1140720/64015481-8c9af880-cb1c-11e9-9b21-a12fee273c8f.png)

I don't know how to do it without modifying the ModalLink component :P But I tried to center it inside a bigger container.  It's looking better now